main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"net"
+	"os"
+
+	pb "github.com/kubearmor/KVMService/protobuf"
+	"google.golang.org/grpc"
+)
+
+type server struct {
+	pb.UnimplementedKVMServer
+}
+
+func (s *server) RegisterAgentIdentity(ctx context.Context, in *pb.AgentIdentity) (*pb.Status, error) {
+	log.Println("Agent Identity is ", in.Identity)
+	os.Setenv("AGENT_IDENTITY", in.Identity)
+	return &pb.Status{Status: 200, ErrorMessage: "nil"}, nil
+}
+
+func main() {
+
+	// Get input flag arguments
+	gRPCPtr := flag.String("gRPC", "32767", "gRPC port number")
```

Comment:
For KVMService, we need to use configmap to keep the configuration.

Reply:
Fine Rahul. Will modify the same.